### PR TITLE
employee register

### PR DESCRIPTION
**Pull request by nsengiyumva-wilberforce**

### CODE REVIEW

File: Client/src/app/auth/register/register.component.html [hunk 1]
```diff
@@ -272,21 +272,23 @@ <h3 class="card-title text-info d-flex justify-content-center">
           >
             Sign up
           </button>
+          
 
           <!-- <ngx-spinner bdColor="#175b6f86" size="medium" color="#27ceaa" type="ball-clip-rotate-multiple" [fullScreen] = "true">
             <h4 class="text-light justify-content-between align-items-center mt-4 pt-4 mt-md-4 pt-md-2">Please wait...</h4>
-          </ngx-spinner> -->
+          </ngx-spinner>
```
Comment (usamaidrsk) [inline Client/src/app/auth/register/register.component.html:279]: why are you commenting out this

File: Client/src/app/auth/register/register.component.ts [hunk 2]
```diff
@@ -136,36 +138,47 @@ export class RegisterComponent implements OnInit {
...
-      //     this.errored = true;
-      //     this.serviceErrors = error;
-      //     this.showDanger();
-      //   }
-      // );
+      console.log(this.appData);
+
+      this.authService.registerUser(this.appData).subscribe(
+        (data: string) => {
+          if (data === "Registration Successfully") {
```
Comment (usamaidrsk) [inline Client/src/app/auth/register/register.component.ts:157]: this response varies with time, you shouldn't statically equate it like this, just check if it exists

File: Client/src/app/auth/register/register.component.ts [hunk 3]
```diff
@@ -136,36 +138,47 @@ export class RegisterComponent implements OnInit {
...
+            this.serviceErrors = "Registration was Successful";
```
Comment (usamaidrsk) [inline Client/src/app/auth/register/register.component.ts:158]: is this an error, yet it reads successful

**Review by usamaidrsk — COMMENTED** [review]
please add screen shots for every test case, like inputs catching errors on data entry, the submit button enabled or disabled where necessary, as well as screens to see the successful message or error message after submitting the data

File: Client/src/app/auth/register/register.component.html (hunk 1 above)
Comment (nsengiyumva-wilberforce) [reply Client/src/app/auth/register/register.component.html:279]: it shows ngx-alerts is not a known element

Comment (usamaidrsk) [reply Client/src/app/auth/register/register.component.html]: okay, i will check on that

**Pull request closed without merging**